azkaban-web-server/src/main/java/azkaban/flowtrigger/DependencyPluginConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.flowtrigger;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class DependencyPluginConfig {
```

Comment:
Why isn't it an interface?

Reply:
I don't see there will be multiple implementations for it.

Reply:
Since we may not have direct control over the plugin code, using an interface gives us more flexibility to evolve this type.

Reply:
updated